src/schemas/21-526EZ/schema.js

```diff
@@ -105,19 +105,40 @@ const fullNameDef = ((definitions) => {
 })(definitions);
 
 /**
- * Strip address lines from PCIU address def for use with treatment center addresses
+ * Modifies PCIU Address for use with treatments schema
  * @typedef {object} definitions
  * @property {object} pciuAddress
  * @param {definitions} definitions from the common schema definitions file
  * @returns {object} the treatmentCenterAddress schema object
  */
-const treatmentCenterAddressDef = ((definitions) => {
-  const treatmentAddressProperties = _.pick(
-    ['city', 'state', 'country'],
-    definitions.pciuAddress.properties
-  );
+const vaTreatmentCenterAddressDef = (({ pciuAddress }) => {
+  const { type, oneOf, properties } = pciuAddress;
+  return Object.assign({}, {
```

Comment:
If we decide to use lodash, could we use _.merge here?

Reply:
I think `merge` merges all properties of a given object from both source and destination which wasn't my intention. That being said, this needs updating anyway since `country` and `state` are things that we need in the VA address and are now included in the common PCIU definition. So 🎉